doc/developer-guides/coding_guidelines.rst

```diff
@@ -3255,3 +3255,56 @@ Compliant example::
        };
 
 
+Implementation-specific Behaviors
+*********************************
+
+
+IB-01: All characters in an identifier are significant initial characters
+=========================================================================
+
+The number of significant initial characters in an identifier is
+implementation-defined, according to J.3.3 item 2 in C99. For ACRN hypervisor,
+all characters in an identifier are significant initial characters.
+
+IB-02: The number of bits in a byte is 8
+========================================
+
+The number of bits in a byte is implementation-defined, according to J.3.4 item
+1 in C99. For ACRN hypervisor, the number of bits in a byte is 8.
+
+IB-03: The values of the members of the execution character set depends on ASCII Table
+======================================================================================
+
+The values of the members of the execution character set is
+implementation-defined, according to J.3.4 item 2 in C99. For ACRN hypervisor,
+characters are encoded in ASCII.
```

Comment:
If you're only allowing ASCII characters in a source file, what are you going to do with lines such as this (found in devicemodel/include/ata.h) where the copyright author's first name has a non-ASCII character in it:
```
 * Copyright (c) 2000 - 2008 Søren Schmidt <sos@FreeBSD.org>
```
Should you amend this to say comments are allowed to have non-ASCII (UTF-8) characters?